Open_Source/Section-5/5.38_Explain_everything_related_to_Flux.md

```diff
@@ -0,0 +1,113 @@
+# Flux
+
+![Flux Image](https://imgur.com/1ZLgq7V.png)
+
+**Flux** is an `architecture` that Facebook uses internally when working with `React`. It is not a framework or a library. It is simply a new kind of **architecture** that complements React and the concept of Unidirectional `Data Flow`.
```

Comment:
please use `<b>` or `<u>` tag to highlight any text instead of using backquotes throughout this file